main.py

```diff
@@ -11,4 +11,5 @@
 print()
 print("Conversions:")
 print("{} kilometres is {} metres".format(num, km_to_m(num)))
+print("{} miles is {} kilometres".format(num, km_to_m(num)))
```

Comment:
You called the wrong function in this line!
```suggestion
print("{} miles is {} kilometres".format(num, miles_to_km(num)))
```
Hope this helps!